app/src/main/java/com/courseproject/tindar/entities/Message.java

```diff
@@ -0,0 +1,63 @@
+package com.courseproject.tindar.entities;
+
+import java.sql.Timestamp;
+
+public class Message {
+    private String messageID;
```

Comment:
can we use messageId instead of messageID so for Id, we will have the second character always in lowercase.

Reply:
I think this is fine. In the methods, ID is spelled properly; in the class we're not doing that so that it doesn't look final.